packages/vue-dot/playground/examples/DialogBoxEx.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+	<DocSection
+		no-divider
+		title="DialogBox"
+	>
+		<h2 class="subtitle-1 mb-4 font-weight-bold">
+			Show list of button actions
+		</h2>
+
+		<VBtn
+			color="primary"
+			class="ml-6 px-5"
+			@click="dialog = true"
+		>
+			Show dialog
+		</VBtn>
+
+		<DialogBox
+			:dialog.sync="dialog"
+			title="DialogBox"
+			@cancel="dialog = false"
+			@validate="dialog = false"
```

Comment:
```suggestion
			@confirm="dialog = false"
```
Est-ce que tu pourrais renommer cet événement en `confirm` stp ? Je trouve cela plus en accord avec `cancel` et c'est un événement déjà utilisé dans des applications